Dockerfile.windows

```diff
@@ -57,70 +57,90 @@ RUN go build -buildmode=c-shared -o out_newrelic.dll .
 #                                                                                                              #
 # -------------------------------------------------------------------------------------------------------------#
 
-#
```

Comment:
All the changes here are actually a result of just copying-pasting the contents of the official Fluent Bit Dockerfile for Windows and pinning the `1.9.4` version instead of `master`: https://github.com/fluent/fluent-bit/blob/master/dockerfiles/Dockerfile.windows